package.json

```diff
@@ -38,6 +40,8 @@
     "recharts": "1.8.5"
   },
   "devDependencies": {
+    "@loadable/babel-plugin": "^5.12.0",
+    "@loadable/webpack-plugin": "^5.12.0",
```

Comment:
This won’t do anything unless it is initialized in a webpack.config.js which would require ejecting